src-plugins/manualRegistration/manualRegistration.cpp

```diff
@@ -58,16 +50,11 @@ class manualRegistrationPrivate
 manualRegistration::manualRegistration() : itkProcessRegistration(), d(new manualRegistrationPrivate)
 {
     d->proc = this;
-    d->registrationMethod = NULL ;
-
-    //set transform type for the exportation of the transformation to a file
-    this->setProperty("transformType","Rigid");
```

Comment:
I don't think you should remove this.
It seems to be used in the *Export last transfo* button